_layouts/packages.html

```diff
@@ -4,16 +4,20 @@
 
 {% capture manual %}{{ site.baseurl }}/Manuals/pkg/{{ page.dirname }}{% endcapture %}
 
-<h2>GAP package {{ page.name }}</h2>
+<h1>GAP package {{ page.name }}</h1>
 
 <h2>{{ page.description }}</h2>
 
+<h4>Short Description</h4>
+<p>{{ page.abstract }}</p>
+
 <p class="homelink">[<a href="{{ page.www }}">WWW homepage</a>]</p>
 
+<p>
```

Comment:
New suggestion: let's use a "description list", plus some CSS, to get this to look nicer
```suggestion
<dl>
```
and then below, instead of `<h4>` / `<b>` we use `<dt>`, and wrap the afterwards into a `<dd>` ... just read up on https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl to get a better explanation.

The result may not look great immediately, *but* it has better semantics (good for people with a screen reader), and now we can easily style it with CSS. Not that I know CSS, but I am sure we can crib some nice styles for that somewhere and just paste them in... :-)

Reply:
https://www.the-art-of-web.com/css/format-dl/